cypress/support/graphql-utils.ts

```
@@ -0,0 +1,156 @@
+  return cy.wait(
+    `@${operationName}`,
+  ) as Cypress.Chainable<Cypress.Interception>;
+};
+
+/**
+ * Mock a GraphQL operation response by operationName.
+ *
+ * Uses interceptGraphQLOperation to route matching requests and applies
+ * the responder based on its shape:
+ * - function: receives the request and can call req.reply/req.continue manually
+ * - string: treated as a fixture path and passed to req.reply({ fixture, ...options })
```

ESLint

[error] tsdoc-malformed-inline-tag: Expecting a TSDoc tag starting with "{@"

(tsdoc/syntax)